### ISSUE TITLE
Handle Broken Images

### ISSUE THREAD

**Issue by RFSH** [informatics-isi-edu/chaise#2771]
Currently if an image is broken or the user doesn't have the required permission to view it, it will show up broken in the browser with its alt text:

<img height="600" alt="Image" src="https://github.com/user-attachments/assets/cab50ebd-32b6-42e7-8f80-f51d1b33aa58" />

We should explore whether we can show messages or default images that help users understand the issue. In the example above, this is happening because the user doesn't have access to it. So we could show a default image to indicate that. Or, for example, if the image is returning a 404, we can show a "Missing" or "Not found" image.

One option is to do this by just adding an `onerror` handler. Otherwise, we would have to send a GET/HEAD request. Similar to the `asset-permission` and `external-link` features, this feature should also be triggered only on special class names.
